core/trino-spi/src/main/java/io/trino/spi/block/Int2IntOpenHashMap.java

```diff
@@ -0,0 +1,287 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.spi.block;
+
+// Note: this code was forked from fastutil (http://fastutil.di.unimi.it/) Int2IntOpenHashMap
+// and mimics that code style.
+// Copyright (C) 2002-2019 Sebastiano Vigna
+class Int2IntOpenHashMap
```

Comment:
I assume this is a direct copy with just code style changes.  If you changed anything else, please highlight those changes